vendor/github.com/docker/docker/vendor.conf

```diff
@@ -6,21 +6,22 @@ github.com/golang/gddo                              72a348e765d293ed6d1ded7b6995
 github.com/google/uuid                              0cd6bf5da1e1c83f8b45653022c74f71af0538a4 # v1.1.1
 github.com/gorilla/mux                              98cb6bf42e086f6af920b965c38cacc07402d51b # v1.8.0
 github.com/Microsoft/opengcs                        a10967154e143a36014584a6f664344e3bb0aa64
-github.com/moby/term                                73f35e472e8f0a3f91347164138ce6bd73b756a9
+github.com/moby/locker                              281af2d563954745bea9d1487c965f24d30742fe # v1.0.1
+github.com/moby/term                                7f0af18e79f2784809e9cef63d0df5aa2c79d76e
```

Comment:
I will update moby/term and golang.org/x/sys separately, as the golang.org/x/sys update is incompatible with containerd/console v1.0.0, and we need to wait for https://github.com/containerd/console/pull/40 to be in a release